librz/core/cmd_analysis.c

```diff
@@ -8088,6 +8088,75 @@ RZ_IPI RzCmdStatus rz_analysis_rtti_demangle_class_name_handler(RzCore *core, in
 	return RZ_CMD_STATUS_ERROR;
 }
 
+RZ_IPI RzCmdStatus rz_analysis_print_global_variable_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	rz_analysis_var_global_list_show(core->analysis, state, argv[1]);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analysis_global_variable_add_handler(RzCore *core, int argc, const char **argv) {
+	const char *var_name = argv[1];
+	ut64 addr = rz_num_math(core->num, argv[2]);
+	const char *type = argv[3];
+
+	RzTypeParser *parser = rz_type_parser_new();
```

Comment:
I think it's better to use `core->analysis->parser` instead, to be able to reuse all currently loaded types